### PR TITLE
chore(colorhandle): migrate mods

### PR DESCRIPTION
**Pull request by cdransf**
## Description

- [x] migrates mod tokens from theme to index

## How and where has this been tested?

Please tag yourself on the tests you've marked complete to confirm the tests have been run by someone other than the author.

### Regression testing

Validate:

1. The documentation pages for at least two other components are still loading, including:

- [ ] The pages render correctly, are accessible, and are responsive.

2. If components have been modified, VRTs have been run on this branch:

- [ ] VRTs have been run and looked at.
- [ ] Any VRT changes have been accepted (by reviewer and/or PR author), or there are no changes.

## To-do list

- [ ] I have read the [contribution guidelines](/.github/CONTRIBUTING.md).
- [ ] I have updated relevant storybook stories and templates.
- [ ] I have tested these changes in Windows High Contrast mode.
- [ ] If my change impacts **other components**, I have tested to make sure they don't break.
- [ ] If my change impacts **documentation**, I have updated the documentation accordingly.
- [ ] ✨ This pull request is ready to merge. ✨

### CODE REVIEW

File: components/colorhandle/index.css [hunk 1]
```diff
@@ -14,6 +14,8 @@
 @import "themes/spectrum-two.css";
 
 .spectrum-ColorHandle {
+	--mod-opacity-checkerboard-position: 50%;
```
Comment (castastrophe) [inline components/colorhandle/index.css:17]:
```suggestion
    /* @passthrough -- opacity checkerboard customization */
	--mod-opacity-checkerboard-position: 50%;
```

This flag will help the linter know to ignore this passthrough customization.

**Review by castastrophe — APPROVED** [review]
Minor linting suggestion; otherwise great!

**Review by castastrophe — APPROVED** [review]

**Comment by castastrophe** [comment]
@cdransf Quick side-note, you can add these as fix'es instead of chores in future since you're making changes to the *.css assets so it will influence the compiled results.

**Comment by cdransf** [comment]
> @cdransf Quick side-note, you can add these as fix'es instead of chores in future since you're making changes to the *.css assets so it will influence the compiled results.

Got it! Will do :)

**Pull request merged**